src/gui/src/AboutDialog.cpp

```diff
@@ -32,6 +32,11 @@ AboutDialog::AboutDialog(QWidget* parent, const QString& app_name) :
     version.append(QStringLiteral("-%1").arg(INPUTLEAP_REVISION));
 #endif
     ui_->m_pLabelAppVersion->setText(version);
+    //logoHeight is the real pixel height of the resource.
+    const int logoHeight = 45;
+    //set the app logo scale to width of the dialog
+    //the scale can range from 1 - 3 x
+    ui_->logoLabel->setPixmap(QPixmap(":/res/image/about.png").scaled(QSize(sizeHint().width(), std::max(logoHeight, std::min(135, (sizeHint().width()/163) * logoHeight))) , Qt::KeepAspectRatio, Qt::SmoothTransformation));
```

Comment:
Can you break that line up please? It's so nested it's hard to decipher.

Also - the magic numbers of 135 and 163 - where do those come from?


/me squints:

I think you can use `std::clamp()` instead of a nested `max(min())`

Reply:
I would have loved to use `std::clamp()` however that is a c++17 feature and we are only needed c++14

Reply:
I renamed them to 
`normalLogoHeight`
`doubleLogoSize`
`scaledLogoSize` 
to better reflect what they should be.